static/templates/settings/stream_specific_notification_row.hbs

```diff
@@ -1,17 +1,20 @@
-<tr class="stream-row" data-stream-id="{{stream.stream_id}}">
+<tr class="stream-row {{#if muted}}control-label-disabled{{/if}}" data-stream-id="{{stream.stream_id}}">
     <td>
         <span id="stream_privacy_swatch_{{stream.stream_id}}" class="stream-privacy filter-icon">
             {{> ../stream_privacy
               invite_only=stream.invite_only
               is_web_public=stream.is_web_public }}
         </span>
         {{stream.stream_name}}
+        {{#if muted}}<div class="fa fa-bell-slash-o unmute_stream" data-tippy-allowHtml="true" data-tippy-content="<b>{{t 'Unmute this stream' }}"></div>{{/if}}
```

Comment:
Instead of not having the element in DOM if stream is not muted, we should hide the element here by adding `display: none` style such that icon is shown if I mute a stream from stream settings.